### DISCUSSION TITLE
IKVM8 future

### DISCUSSION THREAD

**Discussion by sergey-tihon** [sergey-tihon/OpenNLP.NET#10] [General]
Issue for tracking future of IKVM8

> There are some updates to it happening at: https://github.com/wwrd/ikvm8. In particular, there has been some progress on getting IKVM to work with .NET Standard: wwrd/ikvm8#3. You might consider targeting IKVM.WINDWARD instead of IKVM so when these changes are complete this project will support .NET Standard (we will need this for OpenNLP).

Thanks @NightOwl888 for pointing it out

**Comment by ams-tschoening** [comment]
Please note that there's a new organisation covering sources including development history of IKVM and trying to revive things as well:

https://github.com/ikvm-revived
https://github.com/wwrd/ikvm8/issues/11

In my opinion it's better to base any work on IKVM on the fork there.
